pandas/tests/io/test_excel.py

```diff
@@ -1224,25 +1283,31 @@ def test_int_types(self, merge_cells, engine, ext, np_type):
     def test_float_types(self, merge_cells, engine, ext, np_type):
         # Test np.float values read come back as float.
         frame = DataFrame(np.random.random_sample(10), dtype=np_type)
-        frame.to_excel(self.path, 'test1')
+        frame.to_excel(self.path, "test1")
+
         reader = ExcelFile(self.path)
-        recons = read_excel(reader, 'test1').astype(np_type)
+        recons = read_excel(reader, "test1", index_col=0).astype(np_type)
```

Comment:
why is this changed?

Reply:
`index_col=None` was not being respected before, so I need to add `index_col=0` now for this test.